libraries/Microsoft.Bot.Builder.LUIS/LuisModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Microsoft.Cognitive.LUIS;
+
+namespace Microsoft.Bot.Builder.LUIS
+{
+    public class LuisModel : ILuisModel
+    {
+        public LuisModel(string modelId, string subscriptionKey, Uri uriBase, LuisApiVersion apiVersion)
+        {
+            ModelID = modelId;
+            SubscriptionKey = subscriptionKey;
+            UriBase = uriBase;
+            ApiVersion = apiVersion;
+        }
+
+        public string ModelID { get; }
+
+        public string SubscriptionKey { get; }
+
+        public Uri UriBase { get; }
+
+        public LuisApiVersion ApiVersion { get; }
+
+        public double Threshold => 0.0d;
+
+        public LuisRequest ModifyRequest(LuisRequest request)
+        {
+            return request;
```

Comment:
I think I'm missing something obvious. What's the point of this method?

Reply:
I'm working around the annoying way the bot builder v3 Luis Client is implemented, didn't want invest in changing the v3 implementation so much since it's been very stable.